client/src/connection_cache.rs

```diff
@@ -162,82 +125,3 @@ pub fn par_serialize_and_send_transaction_batch(
         Connection::Quic(conn) => conn.par_serialize_and_send_transaction_batch(transactions),
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
I guess we still need some tests even using lru

Reply:
Agreed, though I haven't been able to think of an appropriate test here, since before the tests basically tested whether our home-grown LRU cache did what it was supposed to. Now that we're using the LRU crate, it's more appropriate to rely on the crate's tests for that.

Reply:
I would just add some simple get calls make sure we are using that correctly.